config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -140,9 +141,12 @@ presubmits:
           securityContext:
             privileged: true
           resources:
-            requests:
-              memory: "4Gi"
-              cpu: 2000m
+           requests:
+             cpu: "2"
+             memory: "9Gi"
```

Comment:
Any reason why we went from 4Gi to 9Gi ? we probably want to maintain that to 4Gi.